code/controllers/subsystem/ticker.dm

```diff
@@ -8,7 +8,7 @@ SUBSYSTEM_DEF(ticker)
 	var/static/current_state = GAME_STATE_INIT
 
 	/// Did we attempt an automatic gamemode vote?
-	var/static/auto_gamemode_vote_attempted = FALSE
+	/// fuck this var/static/auto_gamemode_vote_attempted = FALSE
```

Comment:
?

Reply:
I am removing the gamemode vote entirely, and on local I always get the gamemode vote, which is why I commented out (I didnt know I would get rid of the gamemode vote all together)